libmachine/log/bugsnag_test.go

```diff
@@ -0,0 +1,11 @@
+package log
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestBugSnag(t *testing.T) {
```

Comment:
Won't this test fail without network access and/or access to our particular Bugsnag API token?